packages/calcite-components/src/components/accordion-item/accordion-item.scss

```diff
@@ -106,29 +124,27 @@
 }
 
 :host .content {
-  @apply text-color-3 hidden pt-0;
+  @apply hidden pt-0;
   text-align: initial;
 }
 
 // accordion item icon
 :host .expand-icon {
-  @apply text-color-3;
-  margin-inline-start: var(--calcite-accordion-item-icon-spacing-start);
-  margin-inline-end: var(--calcite-accordion-item-icon-spacing-end);
-  transform: rotate(var(--calcite-accordion-item-icon-rotation));
+  margin-inline-start: var(--calcite-internal-accordion-item-icon-spacing-start);
```

Comment:
Nitpick since this is internal, but for consistency, these should use `space` vs `spacing`.

Also, since these are used for inline spacing, would `--calcite-internal-accordion-item-icon-space-x-start/end` be more appropriate?